performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/tools/TraceTemplates.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Can you run the formatter on this code? I am noticing some spacing patterns that are making it difficult to read.